main/getting-started/Documentation-Map.md

```diff
@@ -0,0 +1,69 @@
+- **Getting Started**: These are the first things you should
+read. They cover what to do to install, set up, and start
+using Agoric's smart contract tools. They also introduce those tools and
+their underlying concepts and designs.
+- **Guides**: These are more detailed and complete descriptions
+of our tools and their concepts, designs, and usage. They include
+descriptions and examples of how to do various tasks with Agoric
+software.
+- **Reference**: These are the specs for our APIs and commands,
+  as well as checklist tables that specify and explain the
+  steps for extended procedures, such as installing Agoric
+  software.
+
+Some documents fall into multiple categories. For example, our Agoric
+installation document is both a Reference and a Getting Started document.
+
+You should read them in this order:
+1. **Agoric Prerequisites**: What you need to do before installing
+   Agoric software and installing the Agoric SDK.
+2. **Introduction to ERTP**: A Getting Started document that
+  briefly explains the concepts and components of Agoric's Electronic
+  Rights Transfer Protocol (ERTP) which is the base of Agoric
+  software.
+3. **Introduction to Zoe**: A Getting Started document that
+  briefly explains the concepts and components of Zoe, our platform
+  for writing smart contracts in JavaScript. Zoe implements our smart
+  contract guarantee that users either get what they wanted from the
+  contract or a full refund of what they put up for it. Regardless of
+  whether the contract is honest, buggy, or malicious.
+4. **Using An Agoric Demo Contract**: Agoric supplies several
+  demo smart contracts for you to experiment with, learn from, and
+  even copy code from for use in your projects. This document shows
+  you how to install and modify the Autoswap demo contract.
+5. **ERTP Guide**: A detailed description of ERTP concepts, 
+  design, components, and commands. Includes examples of command 
+  usage.
+6. **Zoe Guide**: A detailed description of Zoe concepts, 
+  design, components, and commands. Includes examples of command 
+  usage.
+7. **Creating New Smart Contracts and Using Existing Ones**: A
+  procedure checklist that walks you through
+  creating a new smart contract from
+  scratch. Another procedure checklist walks you through editing
+  and using an existing contract.
+
+When needed, refer to the **ERTP API Reference**, **Zoe API
+Reference**, and **Agoric CLI Reference** for command specifications. 
+We also have an **Agoric Glossary** for terms we've given Agoric-context
+definitions to (i.e. what does *mint* mean in an Agoric context?).
+
+For more about Agoric's ideas, plans, and goals, see our **Learn More**
+page. It has links to Agoric's company homepage and various documents and 
+videos you may find useful, such as conference talks, white papers, 
+conference papers, etc.
+
+Finally, we have links to the ultimate documentation; the GitHub
+repositories for the code that defines the Agoric SDK and our Zoe
```

Comment:
This division doesn't necessarily make sense to me. ERTP and Zoe are equally part of the Agoric SDK repo. Are we highlighting Zoe for emphasis?